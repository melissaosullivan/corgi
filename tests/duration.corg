duration d = $1/4$;