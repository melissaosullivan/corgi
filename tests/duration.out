
int main()
{
duration d = $1/4$;
}

